extensions/sql-migration/src/constants/strings.ts

```diff
@@ -1590,6 +1594,24 @@ export const MIGRATION_SERVICE_CLEAR = localize('sql.migration.select.service.de
 export const MIGRATION_SERVICE_SELECT_HEADING = localize('sql.migration.select.service.heading', 'Filter the migration list by Database Migration Service');
 export const MIGRATION_SERVICE_SELECT_SERVICE_LABEL = localize('sql.migration.select.service.service.label', 'Azure Database Migration Service');
 export const MIGRATION_SERVICE_SELECT_SERVICE_PROMPT = localize('sql.migration.select.service.prompt', 'Select a Database Migration Service');
+
+// Upload Arm Template Dialog
+export const SELECT_STORAGE_ACCOUNT_TITLE = localize('sql.migration.select.storage.account.title', 'Select Azure Storage Account');
```

Comment:
follow same pattern for all strings. Use "" quotes for string values. Do for other consts as well.

Reply:
done